xml/http_cookies.xml

```diff
@@ -86,10 +86,25 @@
     <param pos="0" name="hw.device" value="VPN"/>
   </fingerprint>
 
-  <fingerprint pattern="^(Apache)=[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.([0-9]+);">
-    <description>Apache</description>
-    <param pos="1" name="cookie"/>
-    <param pos="2" name="system.time.micros"/>
+  <fingerprint pattern="^Apache=(\d{1,3}\.\d{1,3}\.\d{1,3}\.\d{1,3})\.([0-9]+(?:\.[0-9]+)?);">
+    <description>Apache with session ID containing IP and timestamp (timestamp can be micros, millis or seconds)</description>
+    <example host.ip="10.10.130.165" system.time="1643670182768255">Apache=10.10.130.165.1643670182768255; path=/</example>
```

Comment:
This is a fun one:
1. The regular expression was storing the "system time" in `system.time.micros`, but the time stamp that this regular expression was matching comes back from Apache in seconds, milliseconds or microseconds. This seems overly configuration specific. I chose to store the value in `system.time` and leave it to the client application to interpret. A separate option was to explicitly parse microseconds/milliseconds/seconds as separate fingerprints, but this would slow down parsing and run into issues in the future when interpreting these timestamps as epoch timestamps (albeit, a bit far out). 
2. The regular expression didn't catch the "uuid-like" form of apache session cookies, which I added support for.

Reply:
Actually, I think we _can_ and should split these into separate fingerprints per seconds/millis/micros. The integer lengths are far enough apart that it's improbable that these values will ever overlap (a regular expression matching a number of digits should be good enough precision to capture each timestamp).

Reply:
Option 4: Decide we don't really care about the timestamp and ignore it  ;)

Reply:
@tsellers-r7 I'm fine with that too. I'm still not super excited about expanding this into three regular expressions as that'll def slow it down... so changing my mind again, as I don't think the time adds enough value to offset the cost. I think we keep it as one. Happy to either remove the time capture group or leave as is.

Reply:
I don't know any production use of the time value. I'm ok with removing it in order to improve this fingerprint.

CC: @mkienow-r7 @hdm @gschneider-r7 for comments needing to capture time and emit it in the recog results.

Reply:
I received confirmation via Slack from @hdm and @ekelly-rapid7 that we should be good to go removing `system.time` here.